docs/rhoas/cli-context/README.adoc

```diff
@@ -0,0 +1,240 @@
+//OpenShift Streams for Apache Kafka
+:base-url: https://github.com/redhat-developer/app-services-guides/tree/main/docs/
+:product-long-rhoas: OpenShift Application Services
+:rhoas-cli-base-url: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:rhoas-cli-ref-url: commands
+:rhoas-cli-getting-started-url-context: cli-context/
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+:product-kafka: Streams for Apache Kafka
+
+[id="chap-getting-started-rhoas-cli-context"]
+= Getting started with the rhoas CLI for context management
+:context: getting-started-rhoas-context
+
+[role="_abstract"]
+With the {product-long-rhoas} `rhoas` command-line interface (CLI), you can manage service contexts.
+
+As a developer of applications and services,
+you can use `rhoas` to manage service contexts that can be used to generate application configurations.
+Contexts allow you to work with multiple projects using different services easily.
+
+[role="_abstract"]
+A service context is a group of application service instances and their service identifiers.
+By using service contexts, you can group together application service instances that you want to use together.
+
+This guide describes how to get started quickly by doing the following:
+
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-creating-context-cli_getting-started-rhoas-context[Create a service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-setting-services-cli_getting-started-rhoas-context[Set services in service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-generating-config-cli_getting-started-rhoas-context[Generating application configurations]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-switching-context-cli_getting-started-rhoas-context[Switch between contexts]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-unsetting-services-cli_getting-started-rhoas-context[Unset services in service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-deleting-context-cli_getting-started-rhoas-context[Delete a service context]
+* {base-url}{rhoas-cli-getting-started-url-context}#proc-sharing-context-cli_getting-started-rhoas-context[Sharing contexts]
+
+
+[id="ref-kafka-cli-prereqs_{context}"]
+== Prerequisites
+
+[role="_abstract"]
+* You've installed the `rhoas` CLI. For more information, see {base-url}{rhoas-cli-installation-url}[_Installing the rhoas CLI_^].
+
```

Comment:
I would include that user should have kafka and service registry created (their respective getting started)